src/js/components/landing/recentActivity/SubmissionLink.jsx

```diff
@@ -42,6 +42,7 @@ export default class SubmissionLink extends React.Component {
 		if(this.props.type == 'fabs') {
 			content = <a href={"#/FABSAddData/" + this.props.submissionId} className='date-link'>
 						{link}
+						<p>{this.props.submissionId}</p>
```

Comment:
We should label the submissionId somewhere, at first I thought it was a row count